testnet/stacks-node/src/nakamoto_node/stackerdb_listener.rs

```diff
@@ -337,10 +337,8 @@ impl StackerDBListener {
                         block.gathered_signatures.insert(slot_id, signature);
                         block.responded_signers.insert(signer_pubkey);
 
-                        if block.total_weight_signed >= self.weight_threshold {
-                            // Signal to anyone waiting on this block that we have enough signatures
-                            cvar.notify_all();
-                        }
```

Comment:
We could leave this check here, since we only need to wake up on rejections.

Reply:
same as above, i will do another check to see if it is still necessary and eventually i will rollback